NEWS.md

```diff
@@ -242,13 +242,13 @@ details of the default gems or bundled gems.
     * Kernel#caller, Thread::Backtrace::Location’s methods, etc. are also changed accordingly.
 
         Old:
-        ```
+        ```text
```

Comment:
What difference does this change make?

Reply:
It communicates that the syntax highlighting language was intentionally set to none, rather than having been forgotten.

That way, when you search for opening bare triple-back ticks, this won't surface any more.

Reply:
I see. I was so used to <code>```</code> used for plain-text code blocks that I got rather confused by looking at this style as markdown code. It might help some people like you, but it could be rather hard to understand for people like me.

So I don't find it's necessarily a positive change. We probably also don't want to spend time reviewing diffs that change <code>``\`</code> to <code>```text</code>, which does nothing visually and doesn't clarify the intention by itself (at least it didn't to me), going forward. I appreciate your effort to improve the doc but I would like this diff to be removed from the PR.

Reply:
Fair enough, I've pulled it out.

It already served its purpose for me, which was to mark this code block as "fixed" to the regexp I was using to find the triple-back-tick pairs that didn't have a language set.

This could be worth reconsidering in the future, if there was a lint step added to remind people to set the language on new code blocks they write. It's like the diff between `void f()` and `void f(void)`.